### PR TITLE
Make matrix-static POSH (wip)

### PR DESCRIPTION
**Pull request by non-Jedi**
Definitely not done yet since I haven't touched what the actual chat rooms look like, but I though I'd open up the PR so that people can comment and/or start working on CSS if they like.

Let me know if anything looks off.

### CODE REVIEW

**Comment by non-Jedi** [comment]
Note: In the current version, for some reason, the space between `{%d Room.NumJoinedMembers %}` and "members" gets eaten, and I have no clue whatsoever why that is.

**Comment by MTRNord** [comment]
@non-Jedi try {% space %} as it is inside a stripwhitespace (or take a look at the quick template readme)

**Comment by t3chguy** [comment]
yeah within a `{% stripspace %}` block you will need to make spaces explicit: `{% space %}`

If this is annoying within a block, maybe sane to change stripspace -> collapsespace

File: src/github.com/t3chguy/matrix-static/templates/basepage.qtpl [hunk 1]
```diff
@@ -22,8 +22,9 @@ PrintPage prints Page p
         <header>
             {%= p.Header() %}
         </header>
-        <hr>
+        <main>
         {%= p.Body() %}
```
Comment (t3chguy) [inline src/github.com/t3chguy/matrix-static/templates/basepage.qtpl]: should this not be additionally tabbed in then?

**Comment by non-Jedi** [comment]
This branch has diverged enough from the master, that it would require serious re-work to get even close to ready, and honestly I just don't foresee myself having the time/desire to work on this again.

**Pull request closed without merging**